package/CHANGELOG

```diff
@@ -23,6 +23,8 @@ mm/dd/yy richardjgowers, kain88-de, lilyminium, p-j-smith, bdice, joaomcteixeira
   * 0.21.0
 
 Fixes
+  * `PDBWriter` writes unitary `CRYST1` record (cubic box with sides of 1 Å)
+    when `u.dimensions` is `None` (Issue #2679, PR #2685)
```

Comment:
Sorry, just saw this last minute as I was writing the squash merge entry.
This isn't fully true at the moment, could you change this to "when `u.dimensions` is `None` or `np.zero(6)`" (or something to that effect)?